_docs/services/aws-elasticsearchBETA.md

```diff
@@ -89,6 +89,110 @@ For customers that would like to import or export their Elasticsearch data, this
+    es_password=$(echo "${es_credentials}" | jq -r '.password')
+
+    cf ssh my-app -L "9200:${es_hostname}:${es_port}"
+    ```
+**Note**: You'll need to leave this [`cf ssh`]({{ site.baseurl }}{% link _docs/management/using-ssh.md %}) command running and perform the following steps a different terminal so that you can access the remote Elasticsearch instance from your local environment.
+
+* Create a snapshot repository:
+
+    ```sh
+    s3_credentials=$(cf service-key my-s3-bucket my-key | tail -n +3)
+
+    s3_bucket=$(echo "${s3_credentials}" | jq -r '.bucket')
+    s3_region=$(echo "${s3_credentials}" | jq -r '.region')
+    s3_access_key=$(echo "${s3_credentials}" | jq -r '.access_key_id')
+    s3_secret_key=$(echo "${s3_credentials}" | jq -r '.secret_access_key')
+
+    curl -X PUT -u "${es_username}:${es_password}" "localhost:9200/_snapshot/my_s3_repository" -d @<(cat <<EOF
+    {
+      "type": "s3",
+      "settings": {
+        "bucket": "${s3_bucket}",
+        "region": "${s3_region}",
+        "access_key": "${s3_access_key}",
+        "secret_key": "${s3_secret_key}"
+      }
+    }
+    EOF
+    )
+    ```
+
+* Create a snapshot:
+
+    ```sh
+    curl -X PUT -u "${es_username}:${es_password}" "localhost:9200/_snapshot/my_s3_repository/my_s3_snapshot"
+    ```
+
+### Restore from S3 to ES 6.8 and reindex
+This part will vary greatly depending on which language your application is written in and which library is used. 
+1. You need to add the following ES repository to your AWS Elasticsearch Domain
+```
+  {
+    "type": "s3",
+    "settings": {
+      "bucket": "${s3_bucket}",
+      "region": "${s3_region}",
+      "role_arn" : "${snapshotRoleARN}"
+    }
+  }
+```
+
+`s3_bucket` and `s3_region` is same as previous step but the `snapshotRoleARN` is given from service-key credentials for the AWS ES 7.4.
+
+2. Restore using above repository and matching snapshot name in previous step. [ES 6.8 Restore](https://www.elastic.co/guide/en/elasticsearch/reference/6.8/modules-snapshots.html#_restore)
+3. Reindex your index - You will need to do this so the index will be the correct version for when upgrading. [ES 6.8 Reindexing API](https://www.elastic.co/guide/en/elasticsearch/reference/6.8/docs-reindex.html)
+Example reindexing body:
+``` 
+{
+  "source": {
+    "index": "your_index"
+  },
+  "dest": {
+    "index": "your_index_in_6.8"
+  }
+}
+```
+4. Once your indexes are at least version `6.8`, you can move on to the final part!
+### Upgrading ES 6.8 to 7.4
+1. Contact Cloud.gov Support to perform a ES 6.8 to ES 7.4 Upgrade Eligibility
+2. Cloud.gov Support will let you know if your ES 6.8 instance is eligible to upgrade. We will be able to let you know which index is incompatible with ES 7.4. Cloud.gov Support will not have the ability to see why the index is incompatible. 
+3. If is ready, then support can do the upgrade to ES 7.4. This will result in some downtime. 
```

Comment:
```suggestion
3. If the instance is ready to upgrade, then support can do the upgrade to ES 7.4. This will result in some downtime. _Please note that these upgrades and only be done during normal support hours and not schedulable after hours or weekends._
```